ivanti_service_manager/plugin.spec.yaml

```diff
@@ -0,0 +1,1005 @@
+      type: string
+      required: false
+    Category:
+      title: Category
+      description: Category
+      type: string
+      required: false
+    CreatedBy:
+      title: Created By
+      description: Created by
+      type: string
+      required: false
+    CreatedDateTime:
+      title: Created Date Time
+      description: Created date and time
+      type: string
+      required: false
+    JournalType:
+      title: Journal Type
+      description: Journal type
+      type: string
+      required: false
+    LastModBy:
+      title: Last Mod By
+      description: Last modified by
+      type: string
+      required: false
+    LastModDateTime:
+      title: Last Mod Date Time
+      description: Last modified date and time
+      type: string
+      required: false
+    Owner_Valid:
+      title: Owner Valid
+      description: Owner valid
+      type: string
+      required: false
+    Owner:
+      title: Owner
+      description: Owner
+      type: string
+      required: false
+    OwnerTeam_Valid:
+      title: Owner Team Valid
+      description: Owner team valid
+      type: string
+      required: false
+    OwnerTeam:
+      title: Owner Team
+      description: Owner team
+      type: string
+      required: false
+    OwnerType:
+      title: Owner Type
+      description: Owner type
+      type: string
+      required: false
+    ParentLink_Category:
+      title: Parent Link Category
+      description: Parent link category
+      type: string
+      required: false
+    ParentLink_RecID:
+      title: Parent Link Rec ID
+      description: Parent link rec ID
+      type: string
+      required: false
+    ParentLink:
+      title: Parent Link
+      description: Parent link
+      type: string
+      required: false
+    RecId:
+      title: Rec ID
+      description: Rec ID
+      type: string
+      required: false
+    Subject:
+      title: Subject
+      description: Subject
+      type: string
+      required: false
+    TypeAlias:
+      title: Type Alias
+      description: Type alias
+      type: string
+      required: false
+    PublishToWeb:
+      title: Publish To Web
+      description: Publish to web
+      type: boolean
+      required: false
+    AttachmentName:
+      title: Attachment Name
+      description: Attachment name
+      type: string
+      required: false
+    AttachmentExtension:
+      title: Attachment Extension
+      description: Attachment extension
+      type: string
+      required: false
+    AttachmentData:
+      title: Attachment Data
+      description: Notes body
+      type: string
+      required: false
+    ReadOnly:
+      title: Read Only
+      description: Read only
+      type: boolean
+      required: false
+    IsNewRecord:
+      title: Is New Record
+      description: Is new record
+      type: boolean
+      required: false
+    IsUnRead:
+      title: Is Un Read
+      description: Is un read
+      type: boolean
+      required: false
```

Comment:
This is a duplicate, I believe it can be removed.

Reply:
This one is a part of another type - (journal_node), I'll correct the title and description.